pages_builder/assets/javascripts/onready.js

```diff
@@ -15,10 +15,7 @@
   }
 
   if (GOVUK.SelectionButtons) {
-    var selectionButtons = new GOVUK.SelectionButtons('label.selection-button', {
-      'focusedClass' : 'selection-button-focused',
-      'selectedClass' : 'selection-button-selected'
-    });
+    var selectionButtons = new GOVUK.SelectionButtons('label.selection-button');
```

Comment:
Usually the selection button JS [expects to apply to the input](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/docs/javascript.md#selection-buttons), not the label. Is this working correctly?